src/main/java/com/coffee/gifu/config/Constants.java

```diff
@@ -7,11 +7,14 @@
 
     // Regex for acceptable logins
     public static final String LOGIN_REGEX = "^[_.@A-Za-z0-9-]*$";
-    
+
     public static final String SYSTEM_ACCOUNT = "system";
     public static final String DEFAULT_LANGUAGE = "fr";
     public static final String ANONYMOUS_USER = "anonymoususer";
 
+    public static final String RNA_API_PATH = "https://entreprise.data.gouv.fr/api/rna/v1/id/";
```

Comment:
Code Smell ### **Refactor your code to get this URI from a customizable parameter.See Rule**